test/sqlsmith/mzcompose.py

```diff
@@ -152,6 +152,7 @@
     "target replica failed or was dropped",  # expected on replica OoMs with #21587
     "cannot materialize call to",  # create materialized view on some internal views
     "arrays must not contain null values",  # aclexplode, mz_aclexplode
+    "use of AT TIME ZONE with time type is not supported",
```

Comment:
@nrainer-materialize Can you confirm that this is the right approach? As far as I can tell there's no way to disable using `AT TIME ZONE` with the `TIME` type, we just need to ignore the errors. Is that a correct understanding?

Reply:
Yes, I was just about to do the same. Thank you!

Reply:
I have a better fix inside of SQLsmith itself so that we don't generate queries containing `timezone(type)`: https://github.com/MaterializeInc/sqlsmith/commit/0aedb91d707ed5e274ae6050eafa5b67b15cfde8